parser/src/java/org/apache/hadoop/hive/ql/parse/HiveLexerParent.g

```diff
@@ -376,6 +376,8 @@ KW_PKFK_JOIN: 'PKFK_JOIN';
 KW_LEADING: 'LEADING';
 KW_TRAILING: 'TRAILING';
 KW_TRIM: 'TRIM';
+KW_IGNORE: 'IGNORE';
```

Comment:
Are 'ignore' and 'respect' treated as SQL keywords?

Reply:
These are non-reserved keywords: added `Respect` to the list, `Ignore` was already there but it was not defined in the lexer before this patch.